src/Core/MailTemplates/Handlebars/FamiliesForEnterprise/FamiliesForEnterpriseRedeemedToOrgUser.text.hbs

```diff
@@ -0,0 +1,3 @@
+{{#>BasicTextLayout}}
```

Comment:
This file should probably be name 
```
FamiliesForEnterpriseRedeemedToEnterpriseUser
```

Technically, they're both `org` users

Reply:
Renamed